ui/pages/send/send-content/send-gas-row/send-gas-row.container.js

```diff
@@ -98,19 +95,13 @@ function mapDispatchToProps(dispatch) {
   return {
     showCustomizeGasModal: () =>
       dispatch(showModal({ name: 'CUSTOMIZE_GAS', hideBasic: true })),
-    setGasPrice: ({ gasPrice, gasLimit }) => {
+    setGasPrice: ({ gasPrice }) => {
       dispatch(setGasPrice(gasPrice));
       dispatch(setCustomGasPrice(gasPrice));
-      if (gasLimit) {
```

Comment:
setGasTotal sets a key in state that *is never read*, instead 'getGasTotal' returns `gasPrice * gasLimit` from state.